File: Java/src/com/ecc/gkgo1a/K01.java
package com.ecc.gkgo1a;

public class K01 extends IExecutable
{
    @Override
    public void Execute(int num)
    {
        super.Execute(num);
    }

    @Override
    public void Question1()
    {
        System.out.println("Hello World! ようこそJava言語の世界へ！");
    }

    @Override
    public void Question2()
    {
        System.out.println("Hello World!");
        System.out.println("ようこそ");
        System.out.println("Java言語の世界へ！");
    }

    @Override
    public void Question3()
    {
        System.out.println("整数：" + 12345);
        System.out.println("実数：" + 123.456789);
        System.out.println("文字：" + 'X');
        System.out.println("文字列：" + "ABCdef");
    }

    @Override
    public void Question4()
    {
        System.out.println("              ##");
        System.out.println("             #  #");
        System.out.println("             #  #");
        System.out.println("            #    #");
        System.out.println("           #      #");
        System.out.println("         ##        ##");
        System.out.println("       ##            ##");
        System.out.println("    ###                ###");
        System.out.println(" ###       ##    ##       ###");
        System.out.println("##        #  #  #  #        ##");
        System.out.println("##         ##    ##         ##");
        System.out.println(" ##     #            #     ##");
        System.out.println("  ###     ##########     ###");
        System.out.println("     ###              ###");
        System.out.println("        ##############");
    }
}

File: Scala/src/main/scala/PrimaryClasses/K01.scala
package PrimaryClasses
import IExecutable.IExecutable

class K01 extends IExecutable {
  override def Question1(): Unit = {
    println("Hello World!　ようこそScala言語の世界へ！")
  }

  override def Question2(): Unit = {
    println("Hello World!")
    println("ようこそ")
    println("Scala言語の世界へ！")
  }

  override def Question3(): Unit = {
    printf("整数：%d\n", 12345)
    printf("実数：%f\n", 123.456789)
    printf("文字：%c\n", 'A')
    printf("文字列：%s\n", "ABCdef")
  }

  override def Question4(): Unit = {
    println("              ##")
    println("             #  #")
    println("             #  #")
    println("            #    #")
    println("           #      #")
    println("         ##        ##")
    println("       ##            ##")
    println("    ###                ###")
    println(" ###       ##    ##       ###")
    println("##        #  #  #  #        ##")
    println("##         ##    ##         ##")
    println(" ##     #            #     ##")
    println("  ###     ##########     ###")
    println("     ###              ###")
    println("        ##############")
  }
}
